courses/task_four/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+// averageArray that calculate average value
+func averageArray(numbers [6]int) float64 {
+	var sum int = 0
+	for _, value := range numbers {
+		sum += value
+	}
+	return float64(sum) / float64(len(numbers))
+}
+
+// maxStringValue fthat returns the longest word
+func maxStringValue(s []string) (maxVal string) {
+	for _, value := range s {
+		if len(value) > len(maxVal) {
+			maxVal = value
+		}
+	}
+	return
+}
+
+// reverseSlice that returns the reversed copy
+func reverseSlice(s []int64) (reversed []int64) {
+	for i := 0; i < len(s); i++ {
+		reversed = append(reversed, s[len(s)-1-i])
+	} // another way to write a loop
+	return
+}
+
+// printSorted that is sorted values of map by key
+func printSorted(m map[int]string) (s []string) {
+	keys := make([]int, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+
+	sort.Ints(keys)
+
+	for _, k := range keys {
+		s = append(s, m[k])
+	}
+	return
```

Comment:
It should literally print the result, not return by the task description.

Reply:
fixed